recipes/pulse/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pulse" %}
+{% set version = "2019.0" %}
+{% set sha256 = "db2a150e2f6476657642bf7094f6a3912b5841775edbd96c3adbcfa3456c41e6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/finsberg/pulse/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  build:
+    - h5py >=2.8.0
```

Please add python as a dependency
```suggestion
    - h5py >=2.8.0
    - python
```